system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -0,0 +1,129 @@
+//===------------------------- cxa_emscripten.cpp -------------------------===//
+//
+//  Most code in the file is directly copied from cxa_exception.cpp.
+//  TODO(sbc): consider merging them
+//
+//===----------------------------------------------------------------------===//
+
+#include "cxxabi.h"
+#include "cxa_exception.h"
+#include "include/atomic_support.h"
+#include "fallback_malloc.h"
+#include "stdio.h"
+#include "assert.h"
+
+#if 1
```

Comment:
Perhaps instead of `1` this could be a name like `EMSCRIPTEN_DEBUG`? The `1` looks like a mistake, to my eye at least, but I do understand what you said about it. lgtm either way though.